src/test/java/org/apache/commons/io/output/FileWriterWithEncodingTest.java

```diff
@@ -85,6 +85,20 @@ public void constructor_File_existingFile_withContent() throws Exception {
         assertEquals(4, file1.length());
     }
 
+    @Test
+    public void constructorAppend_File_existingFile_withContent() throws Exception {
+        try (FileWriter fw1 = new FileWriter(file1);) {
```

Comment:
The extra semi colon is not needed FYI.

Reply:
ups :-D, have fixed this (also for the other test) - i think you can squash the two commits